### DISCUSSION TITLE
Zed Theme Preview Site

### DISCUSSION THREAD

**Discussion by naturalethic** [zed-industries/zed#10254] [Show and tell]
I've published a site to preview all themes in the zed extensions repository.  It is set to update every 3 hours.

https://zedtheme.com

Feel free to contribute or comment at the repo https://github.com/niarada/zedtheme.com

**Comment by nstlgy** [comment]
This is great! and really useful.

**Comment by o3wiz** [comment]
Very nice!

**Comment by JosephTLyons** [comment]
This is super cool @naturalethic!

**Comment by davccavalcante** [comment]
🟢 Brings the nostalgic essence of vintage green phosphor monitors to your Zed editor. This theme is crafted to offer a unique and immersive visual experience, capturing the original charm of early computing displays. Available in both dark and light versions, the Green Monochrome Monitor Theme transforms your coding environment with its distinct green-on-black and green-on-light-green aesthetics.

https://github.com/Takk8IS/green-monochrome-monitor-theme-for-zed

Green Monochrome Monitor Dark Theme
![screenshot-dark](https://github.com/zed-industries/zed/assets/152639968/fc8aecda-380a-4a27-8318-79c653304ad1)

Green Monochrome Monitor Light Theme
![screenshot-light](https://github.com/zed-industries/zed/assets/152639968/bf375857-4f65-4834-ae67-8f62ad479a42)

Experience the originality and unique aesthetics of the Green Monochrome Monitor Theme and take your coding journey back to the era of early computing displays.

**Reply by JosephTLyons** [reply]
Hey @Takk8IS- this would be really cool to get in a theme extension!

**Reply by davccavalcante** [reply]
Hello I want colleague. Thank you very much for enjoying this project. I will constantly work on it for updates and improvements, but I need the support of the ZED user community. I count on the strength of everyone, this project is for all of us who appreciate this concept.
Later this week, I will publish another version... In fact, in the next few hours. =)

**Comment by MarcPinnell** [comment]
Site appears to be down. Bummer.

**Reply by naturalethic** [reply]
My apologies, it is back up now, thanks for letting me know.

**Reply by ShrirajHegde** [reply]
@naturalethic down again

**Comment by BlackGhost10** [comment]
<img width="1920" height="1080" alt="Screenshotv2" src="https://github.com/user-attachments/assets/ce966e71-06d5-445c-817f-776f65f1ba4d" />
 check this out :)

**Reply by BlackGhost10** [reply]
BlackJv2
